src/Database/TenantDatabaseManagers/SQLiteDatabaseManager.php

```
@@ -129,6 +147,8 @@ public function makeConnectionConfig(array $baseConfig, string $databaseName): a
                 (static::$persistInMemoryConnectionUsing)(new PDO($dsn), $dsn);
             }
         } else {
+            $this->validateDatabaseName($databaseName);
+
             $baseConfig['database'] = database_path($databaseName);
         }
```

PHPMD (2.15.0)

[error] The method makeConnectionConfig uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)